maquetado/scss/titles.scss

```diff
@@ -0,0 +1,25 @@
+$text-big: 24px;
+$text-medium: 16px;
+$text-small: 12px;
+$title-decoration-height: 90px;
+$title-decoration-width: 70px;
+
+.title {
+  font-size: $text-big;
+  margin: 20px;
+  position: relative;
+  padding: 90px;
+
+  &::after {
+    content: '';
+    background-image: url('./assets/triangle.svg');
+    background-repeat: repeat-y;    
+		display: block;    
```

Comment:
you have some indentation issues here. I think you dont need the `display: block` prop